src/views/shared/transaction-actions/transaction-actions.view.tsx

```diff
@@ -9,11 +9,11 @@ import { ReactComponent as TokenSwapIcon } from "src/images/icons/token-swap.svg
 
 interface TransactionActionsProps {
   hideWithdraw: boolean;
-  hideSend: boolean;
-  hideDeposit: boolean;
-  hideSwap: boolean;
-  accountIndex: string;
-  tokenId: number;
+  tokenId?: number;
+  accountIndex?: string;
+  hideSend?: boolean;
+  hideDeposit?: boolean;
+  hideSwap?: boolean;
```

Comment:
I think that now `hideWithdraw` can also be optional, right?

Reply:
Yes but I was thinking that we leave it as required until the moment that we need it as optional

Reply:
It's really optional, but it's not being reflected in the interface. So I would add it also so if anyone use this component in the future he knows that it's optional.